### ISSUE TITLE
Waypoint Group Move Glitch

### ISSUE THREAD

**Issue by labtoast** [diydrones/apm_planner#647]
When moving a group of waypoints, all waypoints not under the mouse when dragging will pop back to the original location after clicking away to deselect group.

To recreate
1. Open apm_planner
2. create 3 waypoints
3. ctrl+click all waypoints to select all
4. drag a waypoint to new location
5. click empty space to deselect waypoints

Two waypoints will jump to a different location.

Platform:
OSX 10.10.2, Qt5.3.2, apm_planner master branch

**Comment by malcom2073** [comment]
Yep, looks like only the clicked waypoint is getting finalized, thanks for the report.
